internal/datanode/data_sync_service.go

```diff
@@ -192,7 +192,7 @@ func (dsService *dataSyncService) initNodes(vchanInfo *datapb.VchannelInfo) erro
 			zap.Int64("SegmentID", fs.GetID()),
 			zap.Int64("NumOfRows", fs.GetNumOfRows()),
 		)
-		if err := dsService.replica.addFlushedSegment(fs.GetID(), fs.CollectionID,
```

Comment:
Wouldn't it be better to use "Timestamp" as the parameter if the only usage of `vchanInfo` in replica is  `vchanInfo.GetSeekPosition().GetTimestamp()` ?

Maybe name it into "recoverTs"?

Reply:
done